src/lib/containers/HasAccess.tsx

```diff
@@ -321,13 +316,47 @@ export default class HasAccess extends React.Component<
     )
   }
 
+  renderOpenAccessIcon() {
+    return (
+      <ThemeContext.Consumer>
+        {theme => {
+          return (
+            <IconSvg
+              options={{
+                icon: 'accessOpen',
+                color: theme.colors.success,
+              }}
+            />
+          )
+        }}
+      </ThemeContext.Consumer>
+    )
+  }
+
+  renderLockedIcon() {
```

Comment:
would it make sense to create more generic component that takes color type ('success', 'warning' etc) and icon?

Reply:
At the very least, I can definitely combine the two new methods that I wrote. It's a little weirder to combine with the existing code since we're using FontAwesome for the other icons

Reply:
Yes - just the new ones.